pkg/operator/cleanup/cleanup_controller_test.go

```diff
@@ -287,7 +287,7 @@ func testPassthroughCredentialsRequest(t *testing.T) *minterv1.CredentialsReques
 		ObjectMeta: metav1.ObjectMeta{
 			Name:        testStaleCRName,
 			Namespace:   testNamespace,
-			Finalizers:  []string{minterv1.FinalizerDeprovision},
+			Finalizers:  []string{},
```

Comment:
Rather than this, can we leave the Finalizer, but change the codepath that verifies deltion from assert.Nil() to assert.NotNil(t, cr.DeletionTimestamp, ...) specifically:
```
-                               assert.Nil(t, cr, "expected credentials request to be deleted")
+                               assert.NotNil(t, cr.DeletionTimestamp, "expected credentials request to be marked for deletion")
```

Reply:
Absolutely, updated